reference/commands/list.rst

```diff
@@ -1,95 +1,109 @@
+      -v [V]                Level of detail of the output. Valid options from less verbose to more verbose: -vquiet, -verror,
+                            -vwarning, -vnotice, -vstatus, -v or -vverbose, -vv or -vdebug, -vvv or -vtrace
+      --logger              Show the output with log format, with time, type and message.
+      -p PACKAGE_QUERY, --package-query PACKAGE_QUERY
+                            Only list packages matching a specific query. e.g: os=Windows AND (arch=x86 OR compiler=gcc)
+      -r REMOTE, --remote REMOTE
+                            Remote names. Accepts wildcards
+      -c, --cache           Search in the local cache
+
+
+The ``conan list`` command is an in-depth-oriented command, and it can list recipes and packages
+from the local cache or from any specified remotes. Depending on the patterns specified as argument,
+it is possible to list:
+
+* Recipe references.
+* Recipe revisions.
+* Package IDs and their configurations.
+* Package revisions.
+
+Use cases
+---------
+
+Show recipe references
+**********************
 
 .. code-block:: bash
 
-    $ conan list recipes zlib -r=conancenter
-    conancenter:
-    zlib
+    $ conan list *
+    Local Cache:
+      hello
+        hello/1.0
+        hello/2.0
+      string-view-lite
+        string-view-lite/1.6.0
+      zlib
         zlib/1.2.11
-        zlib/1.2.8
 
-    $ conan list recipes zlib/1.2.1* -r=conancenter
-    conancenter:
-    zlib
+
+.. code-block:: bash
+
+    $ conan list zlib/*
+    Local Cache:
+      zlib
         zlib/1.2.11
 
-    $ conan list recipes zlib/1.2.1* -r=conancenter --format=json
-    [
-        {
-            "remote": "conancenter",
-            "error": null,
-            "results": [
-                {
-                    "name": "zlib",
-                    "id": "zlib/1.2.11"
-                }
-            ]
-        }
-    ]
-
-conan list package-ids
-----------------------
+Show recipe revisions
+*********************
 
 .. code-block:: bash
 
-    $ conan list package-ids zlib/1.2.11 -r=conancenter
-    ...
-    zlib/1.2.11:1513b3452ef7e2a2dd5f931247c5e02edeb98cc9
-        settings:
-        os=Macos
-        arch=x86_64
-        compiler=apple-clang
-        build_type=Debug
-        compiler.version=10.0
-        options:
-        shared=False
-        fPIC=True
-    zlib/1.2.11:963bb116781855de98dbb23aaac41621e5d312d8
-        settings:
-        os=Windows
-        compiler.runtime=MTd
-        arch=x86_64
-        compiler=Visual Studio
-        build_type=Debug
-        compiler.version=15
-        options:
-        shared=False
-    zlib/1.2.11:bf6871a88a66b609883bce5de4dd61adb1e033a7
-        settings:
-        os=Linux
-        arch=x86_64
-        compiler=gcc
-        build_type=Debug
-        compiler.version=5
-        options:
-        shared=True
-    ...
-
-
-
-conan list recipe-revisions
----------------------------
+    $ conan list zlib/1.2.11#*
```

Comment:
I think maybe we should explain a bit here or at least link to other part of the documentation about the different parts that compose a Conan reference: `name/version@use/channel#rrev:pkgid#prev`

Reply:
Good point!